factory_bot.gemspec

```diff
@@ -1,18 +1,18 @@
 $LOAD_PATH << File.expand_path("../lib", __FILE__)
-require 'factory_bot/version'
+require "factory_bot/version"
 
 Gem::Specification.new do |s|
-  s.name        = %q{factory_bot}
+  s.name        = "factory_bot"
   s.version     = FactoryBot::VERSION
-  s.summary     = %q{factory_bot provides a framework and DSL for defining and
-                      using model instance factories.}
-  s.description = %q{factory_bot provides a framework and DSL for defining and
-                      using factories - less error-prone, more explicit, and
-                      all-around easier to work with than fixtures.}
+  s.summary     = "factory_bot provides a framework and DSL for defining and "\
+                  "using model instance factories."
+  s.description = "factory_bot provides a framework and DSL for defining and "\
+                  "using factories - less error-prone, more explicit, and "\
+                  "all-around easier to work with than fixtures."
 
-  s.files         = `git ls-files`.split("\n").reject { |f| f.match(%r{^(spec|features)/}) }
+  s.files = Dir.glob("lib/**/*") + %w[GETTING_STARTED.md LICENSE README.md]
```

Comment:
going to include NEWS and CONTRIBUTING.md as well (and in https://github.com/thoughtbot/factory_bot_rails/pull/286)